src/DynamoCoreWpf/Views/CodeBlocks/CodeBlockEditorUtils.cs

```diff
@@ -134,8 +139,12 @@ public static void CreateHighlightingRules(ICSharpCode.AvalonEdit.TextEditor edi
             var rules = editor.SyntaxHighlighting.MainRuleSet.Rules;
 
             rules.Add(CodeHighlightingRuleFactory.CreateNumberHighlightingRule());
-            rules.Add(CodeHighlightingRuleFactory.CreateClassHighlightRule(controller));
-            rules.Add(CodeHighlightingRuleFactory.CreateMethodHighlightRule(controller));
+            
+            var classRule = CodeHighlightingRuleFactory.CreateClassHighlightRule(controller);
```

Comment:
Do not create rules if there are no loaded types or methods, for example in test scenarios.